packages/react/src/elements/components/Banner/BannerPresenter/styled.js

```diff
@@ -0,0 +1,219 @@
+/* eslint-disable react/prop-types */
```

Comment:
Why `styled.js`?

This file helps maintain the separation between style and logic. This module is formatted in a way that makes it much easier to migrate to CSS modules or a CSS-in-JS solution.

Ignore the jsDoc for now, it's repetitive and verbose, and I'll simplify it later.